packages/fx-core/resource/yaml-schema.json

```diff
@@ -0,0 +1,1495 @@
+    },
+    "aadAppCreate": {
+      "type": "object",
+      "oneOf": [
+        { "$ref": "#/definitions/aadAppCreateWithoutSecret" },
+        { "$ref": "#/definitions/aadAppCreateWithSecret" }
+      ]
+    },
+    "aadAppUpdate": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Update Azure Active Directory application. Please refer to https://aka.ms/teamsfx-actions/aadapp-update for more details.",
+      "required": ["uses", "with"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "An optional name of this action"
+        },
+        "env": {
+          "type": "object",
+          "description": "Define environment variables",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "uses": {
+          "type": "string",
+          "description": "Update Azure Active Directory application based on the given Azure Active Directory application manifest. If the manifest uses AAD_APP_ACCESS_AS_USER_PERMISSION_ID and the environment variable is empty, this action will generate a random id and output it. Please refer to https://aka.ms/teamsfx-actions/aadapp-update for more details.",
+          "const": "aadApp/update"
+        },
+        "with": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "Parameters for this action",
+          "required": ["manifestPath", "outputFilePath"],
+          "properties": {
+            "manifestPath": {
+              "type": "string",
+              "description": "Path of Azure Active Directory application manifest. Environment variables in the manifest will be replaced before applying the manifest to Azure Active Directory application."
+            },
+            "outputFilePath": {
+              "type": "string",
+              "description": "Generate the final Azure Active Directory application manifest used to update Azure Active Directory application to this path."
+            }
+          }
+        }
+      }
+    },
+    "armDeploy": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Execute ARM deployment. Refer to https://aka.ms/teamsfx-actions/arm-deploy for more details on the naming convertion rule.",
+      "required": ["uses", "with"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "An optional name of this action"
+        },
+        "env": {
+          "type": "object",
+          "description": "Define environment variables",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "uses": {
+          "type": "string",
+          "description": "Execute ARM deployment. Every bicep output will be persisted in current environment's .env file with certain naming convertion. Refer to https://aka.ms/teamsfx-actions/arm-deploy for more details on the naming convertion rule.",
```

Comment:
"Create Azure resources using the referenced Bicep / JSON files. Outputs from Bicep / /JSON will be persisted in the current Teams Toolkit environment. Refer to ..."